internal/third_party/deepcopy/util.go

```diff
@@ -0,0 +1,39 @@
+package deepcopy
+
+// DeepCopyMap a function for deep copying map[string]interface{}
+// values. Inspired by:
+// https://gist.github.com/soroushjp/0ec92102641ddfc3ad5515ca76405f4d by Soroush Pour
+// The gist code is MIT licensed
+// which in turn has been inspired by the StackOverflow answer at:
+// http://stackoverflow.com/a/28579297/1366283
+//
+// Uses the golang.org/pkg/encoding/gob package to do this and therefore has the
+// same caveats.
+// See: https://blog.golang.org/gobs-of-data
+// See: https://golang.org/pkg/encoding/gob/
+
+import (
+	"bytes"
+	"encoding/gob"
+)
+
+func init() {
+	gob.Register(map[string]interface{}{})
+}
+
+// DeepCopyMap performs a deep copy of the given map m.
```

Comment:
Comment needs to be updated to match the new function signature

Reply:
Oops, yes, will do 🙈

Reply:
Weird that ci passed :thin

Reply:
Created an issue for that #6683

Reply:
Fixed this! 😄